apps/ima/views/partials/arrival-date-detail.html

```diff
@@ -0,0 +1,21 @@
+<fieldset id="arrival-date-detail-fieldset" class="govuk-radios__conditional" aria-hidden="false">
+    <div
+    class="govuk-form-group govuk-date-input {{#errors.arrived-date}}govuk-form-group--error{{/errors.arrived-date}}">
+        <fieldset>
+            <legend class="govuk-fieldset__legend">
+                <strong>{{#t}}fields.arrived-date.legend{{/t}}</strong>
+                <span id="arrived-date-hint" class="govuk-hint">
+                {{#t}}fields.arrived-date.hint{{/t}}
+                </span>
+                {{#errors.arrived-date}}
+                <p class="govuk-error-message">
+                {{errors.arrived-date.message}}
+                </p>
+                {{/errors.arrived-date}}
+            </legend>
+            {{#input-date}}arrived-date{{/input-date}}
+        </fieldset>
```

Comment:
Initially had it as renderField but this caused a problem where, if there was a validation error on that field, and the user changes the main radio button option and presses 'Continue', it would not clear the validation, leaving the user stuck on the page. Writing out the fieldset solved this problem.